code/_onclick/other_mobs.dm

```diff
@@ -194,7 +194,7 @@
 
 /mob/living/simple_animal/hostile/UnarmedAttack(atom/A)
 	target = A
-	if(dextrous && !is_type_in_typecache(A, environment_target_typecache) && !ismob(A))

```

Comment:
this means simple animals with hands that can also attack objects will be unable to attack objects.

Reply:
Well I mean dextrous mobs already couldn't attack most object types, a really elegant solution would probably be giving dextrous mobs an intent button that lets them decide if they want to use a computer or smash it... but it seems like to most straightforward solution here is to throw in a !(isstructure(A)) condition here so that you preserve like 90% of the old behavior.